services/web/server/src/simcore_service_webserver/director_v2_handlers.py

```diff
@@ -26,42 +31,65 @@
 @permission_required("project.read")
 @log_decorator(logger=log)
 async def start_pipeline(request: web.Request) -> web.Response:
-    director2_settings: Directorv2Settings = get_settings(request.app)
+    client = DirectorV2ApiClient(request.app)
+
+    run_policy = get_run_policy(request.app)
+    assert run_policy  # nosec
+
+    user_id = UserID(request[RQT_USERID_KEY])
+    project_id = UUID(request.match_info["project_id"])
 
-    user_id = request[RQT_USERID_KEY]
-    project_id = request.match_info.get("project_id", None)
     subgraph: Set[str] = set()
-    force_restart = False
+    force_restart: bool = False
```

Comment:
is that really needed?

Reply:
just keeping what was there. I will mark is as deprecated and will remove next iteration